.github/workflows/on_pull_request.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Install dependencies
-        run: python3 -m pip install tox
+        run: python3 -m pip install tox==3.27.1
```

Comment:
I think the tox version doesn't need to be pinned if you fix [this](https://github.com/canonical/kratos-operator/pull/22/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R39) in `tox.ini` file